src/api/app/models/staging/staging_project.rb

```diff
@@ -112,6 +112,11 @@ def current_missing_checks
       (relevant_status_reports + relevant_status_reports_for_architectures).map(&:missing_checks).flatten
     end
 
+    def failed_status_checks
+      @failed_status_checks ||= relevant_checks.where(state: Status::Check::FAILED_STATES) +
```

Comment:
so @ChrisBr there is no merge conflict per se, but relevant_checks no longer exists

Reply:
Updated

Reply:
but not pushed? :)